src/drivers/vmount/input_mavlink.cpp

```diff
@@ -285,7 +285,7 @@ int InputMavlinkCmdMount::update_impl(unsigned int timeout_ms, ControlData **con
 
 void InputMavlinkCmdMount::_ack_vehicle_command(uint16_t command)
 {
-	vehicle_command_ack_s vehicle_command_ack;
+	vehicle_command_ack_s vehicle_command_ack = {};
```

Comment:
This is forcing an initialisation on every iteration - is this really required? Benchmarking PX4 we found that the system is spending quite a bit of time in these defensive programming patterns. Which are generally good, but induce a lot of load (5-8% CPU IIRC) in embedded.